.babelrc

```diff
@@ -1,3 +1,11 @@
 {
-  "plugins": ["transform-react-jsx"]
+  "plugins": ["transform-react-jsx"],
+  "env": {
+    "development": {
```

Comment:
Don't we need production as well?

Reply:
My goal here was only to transpile it for the tests only. It was not my intention to provide illusion of supporting old browsers (because at this stage we are not really supporting old browsers)